code/modules/clothing/under/jobs/engineering.dm

```diff
@@ -40,30 +40,37 @@
 	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON
 
 /obj/item/clothing/under/rank/engineering/engineer
-	desc = "It's an orange high visibility jumpsuit worn by engineers. It has minor radiation shielding."
-	name = "engineer's jumpsuit"
+	desc = "It's an orange high visibility jumpsuit worn by mechanics. It has fire resistance."
+	name = "mechanic's jumpsuit"
 	icon_state = "engine"
-	item_state = "engi_suit"
+	item_state = "b_suit"
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 10, "fire" = 60, "acid" = 20)
 	resistance_flags = NONE
 
-/obj/item/clothing/under/rank/engineering/engineer/hazard
-	name = "engineer's hazard jumpsuit"
-	desc = "A high visibility jumpsuit made from heat and radiation resistant materials."
-	icon_state = "hazard"
-	item_state = "suit-orange"
-	alt_covers_chest = TRUE
+/obj/item/clothing/under/rank/engineering/engineer/nt
+	desc = "It's an yellow high visibility jumpsuit worn by engineers. It has minor radiation shielding."
```

Comment:
You should specify that this is a Nanotrasen uniform somewhere in the description. Probably just "worn by Nanotrasen engineers" or something along those lines.